_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+Below that you will see a bunch of vertical bars, for now we will leave these alone until we have audio pushing through them. For that, we will start by pushing computer audio to them through a digital inputs.
+
+
+### Digital Inputs and Windows Audio
+For digital inputs we will be looking at the middle columns of Voicemeeter Banana which should look a little something like this:
+
+Voicemeeter Digital Inputs:
+
+![https://i.imgur.com/pkMT14v.png](https://i.imgur.com/pkMT14v.png)
+
+To set our Windows Default Audio to push through these digital inputs we need to open Windows Sound Settings by right clicking on the speaker icon that adjusts our audio in the task bar. After right clicking select 'Open Sound Settings' and you should see a window with
+
+Sound-
+Choose your output device:
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+Select VoiceMeeter Input (VB-Audio Voicemeeter VAIO) as the default
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+After setting VB-Audio Voicemeeter VAIO as our default output device pull up Voicemeeter Banana. In the middle under the Voicemeeter VAIO section we should see the audio levels rising and lowering if you play sounds on our computer. Do you? Great! If not? Try again!
+
+You may or may not be able to hear these right away, if not just take a look in the column down at the A1, A2, A3 and select the ones for the headphones. If you set up along with the tutorial it should be A1. At this point you should be able to hear the audio going through your headphones which will show in the bottom right level bars under the A1 column.
+
+Bottom right Hardware Outputs audio with audio pushed through.
+
+You will notice this has a bunch of different options, including a mono toggle and a mute toggle! At this point you should be hearing your computer audio going through and being pushed to your headphones.
+
+
+### Hardware inputs and microphone
+Are you ready to set up a microphone? I am! Alright, so microphones. The far left section, Hardware Inputs, there should be a 'Select your device' listing, click this and you should get an option between a bunch of different things. Select the WDM version of your microphone for now, we can get into different versions of a device driver later but for now just use WDM.
+
+Picture of a microphone selected in Hardware Inputs section:
+
+You will now notice if your microphone is unmuted it should be moving the audio bar under that device listing. If you want to hear what is going through the microphone you can push the A corresponding to your headphones, if you set them up following this tutorial then they should be A1. Can you hear yourself? Good! If not, something might be wrong. if you see no audio being bounced in the level meter alongside your hardware input, windows audio settings might be weird and you may have to adjust your audio settings within the audio driver for that device in Windows Recording Devices.
```

Comment:
- Capitalize "if".
- For some reason, the last sentence feels like we don't explicitly know when something is wrong, what to look for exactly, or how to fix it. I'm wondering if this could be rewritten to be more direct and clear on what exactly to look for and do.

Reply:
Revised this to:  

If you set them up following this tutorial then they should be A1. Can you hear yourself? Good! If not, make sure your microphone doesn't have a physical mute toggle on it and that it isn't muted. If that isn't the issue, you can also check windows sound settings under Recording Devices to see if the input level is too low. Windows can sometimes default audio levels on updates.